debian/control

```diff
@@ -38,7 +38,8 @@ Build-Depends: cdbs (>= 0.4.41),
 	       libwebkit2gtk-4.0-dev,
 	       libudisks2-dev,
 	       liblzma-dev,
-	       libattr1-dev
+	       libattr1-dev,
+	       gnupg
```

Comment:
Nitpick: Could add a trailing comma here to avoid diff noise in future.

Also, nitpick: Would be nice to be in alphabetical order.

Reply:
Most of these will go away with https://phabricator.endlessm.com/T17082 which seems a good time to sort them rather than introducing noise now. I'll add the comma though!